tools/run_tests/artifacts/distribtest_targets.py

```diff
@@ -238,7 +238,7 @@ def __init__(self, platform, arch, docker_suffix=None):
         self.platform = platform
         self.arch = arch
         self.docker_suffix = docker_suffix
-        self.labels = ['distribtest', 'php7', platform, arch]
+        self.labels = ['distribtest', 'php', 'php7', platform, arch]
```

Comment:
That's a misunderstanding. This PR has nothing to do with PHP5 and it doesn't add any distribtest. It simply adds a new label 'php', so that running PHP distribtests from commandline becomes consistent with building PHP artifacts.

E.g.  you can run `tools/run_tests/task_runner.py -f artifact linux php`, to build all the PHP artifacts (they are all PHP7) but  to run distribtests you'd need to do `tools/run_tests/task_runner.py -f artifact linux php7`, which seemed counterintuitive to me. So I adjusted the labels so that running all PHP distribtests can also be done with
`tools/run_tests/task_runner.py -f artifact linux php`, not just `tools/run_tests/task_runner.py -f artifact linux php7`.

This change isn't necessary, but I thought it makes stuff less confusing.

Reply:
I see, thanks for clarifying. So this only makes sure we run all the tests (which are already PHP 7 in content), but have inconsistent labels. Got it. thanks.